### PR TITLE
Add openvswitch ssl test for SLE FIPS/Tumbleweed

### PR DESCRIPTION
**Pull request by mitiao**
Verified runs:
sle fips: http://10.67.17.71/tests/169
tumbleweed: http://10.67.17.71/tests/170

### CODE REVIEW

File: tests/console/openvswitch_ssl.pm [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    # Get pox for openflow test
+    assert_script_run 'wget --quiet ' . data_url('pox.tar.bz2');
+    assert_script_run 'tar jvfx pox.tar.bz2';
+
+    # Setup a simulated open-flow controller with POX
+    type_string
+"pox/./pox.py openflow.of_01 --port=6634 --private-key=/etc/openvswitch/server-key.pem --certificate=/etc/openvswitch/server-cert.pem --ca-cert=/etc/openvswitch/ca-cert.pem 2>&1 | tee /dev/$serialdev &\n";
+    # Wait seconds for pox up
+    wait_still_screen 10;
```
Comment (okurz) [inline tests/console/openvswitch_ssl.pm]: Why not just the wait_serial?

File: tests/console/openvswitch_ssl.pm [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    # Set controller for the vswitch
+    assert_script_run "ovs-vsctl set-controller ovsbr0 \"ssl:127.0.0.1:6634\"";
+
+    # Wait a moment for connection and check it
+    wait_still_screen 15;
+    type_string "ovs-vsctl show | tee /dev/$serialdev\n";
+    die "connection failed" unless (wait_serial qr/Controller "ssl:127\.0\.0\.1:6634".*is_connected: true/ms);
+
+    # Stop pox
+    script_run "ps aux|grep '[p]ox.py'|awk '{print \$2}'|xargs kill";
```
Comment (okurz) [inline tests/console/openvswitch_ssl.pm]: This should not fail as well so assert_script_run?

File: tests/console/openvswitch_ssl.pm [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    # Set SSL for openvswitch and connect to open-flow controller (POX)
+    assert_script_run 'ovs-vsctl set-ssl /etc/openvswitch/client-key.pem /etc/openvswitch/client-cert.pem /etc/openvswitch/ca-cert.pem';
+
+    # Set controller for the vswitch
+    assert_script_run "ovs-vsctl set-controller ovsbr0 \"ssl:127.0.0.1:6634\"";
+
+    # Wait a moment for connection and check it
+    wait_still_screen 15;
```
Comment (okurz) [inline tests/console/openvswitch_ssl.pm]: This also looks a bit unsafe. How about checking the next two lines in a loop with a sleep after each not successful call?

**Comment by mitiao** [comment]
Fixed

**Pull request merged**